pyproject.toml

```diff
@@ -175,8 +175,7 @@ linting = [
 test = [
   "covdefaults",
   "pytest",
-  "pytest-asyncio; python_version < \"3.13\"",
-  "pytest-asyncio @ git+https://github.com/provinzkraut/pytest-asyncio.git@shutdown-asyncgens; python_version >= \"3.13\"",
+  "pytest-asyncio",
```

Comment:
Should probably add a lower bound for the latest release here?

Reply:
It looks like they removed support for python 3.8, so I don't think we can pin it to the latest version that was just released for all pythons.  Here's the lock file entry:
```py
    { name = "pytest-asyncio", version = "0.24.0", source = { registry = "https://pypi.org/simple" }, marker = "python_full_version < '3.9'" },
    { name = "pytest-asyncio", version = "0.25.2", source = { registry = "https://pypi.org/simple" }, marker = "python_full_version >= '3.9'" },
    ```

Reply:
Could we pin for >=3.13 only at least? I just want to avoid some accidental downgrades in the future

Reply:
pinned the newer version for 3.9+ and pinned the last supported version for 3.8